jetty-http2/http2-common/src/main/java/org/eclipse/jetty/http2/HTTP2Flusher.java

```diff
@@ -365,7 +365,7 @@ protected void onCompleteFailure(Throwable x)
         // If the failure came from within the
         // flusher, we need to close the connection.
         if (closed == null)
-            session.abort(x);
+            session.onWriteFailure(x);
```

Comment:
comment is not correct as you are not directly closing.
How is onWriteFailure different to abort?